### PR TITLE
Update readme to include docker-compose generation steps

### PR DESCRIPTION
**Pull request by SebastianMaj**
**Issue**

- Link: https://github.com/NebraLtd/helium-miner-software/pull/272
- Summary: Switched to generate a docker-compose.yml file based on a jinja2 template with a settings.ini file

**How**
Added readme steps for generating the docker-compose.yml file

Closes: #340

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Repo: [github.com/balenablocks/dbus](https://github.com/balenablocks/dbus)
 
 This container configures a DBus session bus instance that is used by the gateway config container to communicate with the miner code (note that there is also a separate system bus running on the host OS which is used by gateway config to communicate with bluez for configuring Bluetooth services). This removes the need to have a custom `com.helium.Miner.conf` dbus config file on the host OS as was done previously (and meant we had to run a custom balena device type).
 
+
```
Comment (shawaj) [inline README.md:47]: Extra blank line?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Helium Miner Software
 
-This repository includes the main [docker-compose.yml](https://github.com/NebraLtd/helium-miner-software/blob/master/docker-compose.yml) file that powers the Nebra miners.
+This repository includes the main docker-compose.yml(Follow the steps to generate this [here](#generating-docker-compose-file)) file that powers the Nebra miners.
```
Comment (shawaj) [inline README.md:3]: Missing space before bracket

Also could change includes to generates to further clarify

**Comment by shawaj** [comment]
Thanks for sorting this @SebastianMaj very helpful 👍

File: README.md (hunk 2 above)
Comment (SebastianMaj) [reply README.md]: Updated the text here, sounds a lot better with generates instead of includes

File: README.md (hunk 1 above)
Comment (SebastianMaj) [reply README.md]: Ah indeed, I must've added an extra one on accident. Good catch!

File: README.md [hunk 3]
```diff
@@ -54,6 +55,26 @@ This is a guide to help you get started with the repository and get it running o
 - Git installed - [download here](https://git-scm.com/downloads)
 - [Balena CLI](https://github.com/balena-io/balena-cli) (Install located on Balena step in quick start steps below)
 
+### Generating Docker Compose File
+
```
[the remark this answers is not in the document]
Comment (SebastianMaj) [reply README.md:58]: Done, I mentioned that latest docker-compose files used in production can be found in the folder.

File: README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Helium Miner Software
 
-This repository includes the main [docker-compose.yml](https://github.com/NebraLtd/helium-miner-software/blob/master/docker-compose.yml) file that powers the Nebra miners.
+This repository generates the main docker-compose.yml(Follow the steps to generate this [here](#generating-docker-compose-file)) file that powers the Nebra miners.
```
Comment (shawaj) [inline README.md:3]: still no space here?

```
the main docker-compose.yml(Follow the steps
```

Comment (shawaj) [reply README.md]: fixed

**Review by shawaj — APPROVED** [review]

**Pull request merged**